documentation/workflows.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: sub-navigation
+order: 9
+title: Workflows
+related:
+  sections:
+    - title: Related links
+      items:
+        - text: Screens
+          href: ../screens
+---
+
+The concept of a workflow is key to understanding how a case on DECS is worked.
+
+## What is a DECS workflow?
+A workflow on DECS is defined using Business Process Model and Notation, and is orchestrated in the application by [Camunda](https://camunda.com/).
+
+> If you are unfamiliar with BPMN and Camunda, [follow a tutorial](https://camunda.com/bpmn/) to get a basic understanding of the principles.
+
+The workflows are defined in and controlled by the [`hocs-workflow` service](https://github.com/UKHomeOffice/hocs-workflow/).
```

Comment:
It would be useful to have a pointer to where the `.bpmn` files are here - either an extra link to https://github.com/UKHomeOffice/hocs-workflow/tree/main/src/main/resources/processes or update the hocs-workflow readme to document that.

Reply:
Added